docs/CLI.md

```diff
@@ -285,6 +285,9 @@ For example, suppose you want to run only tests related to authorization which
 will have names like `"GET /api/posts with auth"`, then you can use
 `jest -t=auth`.
 
+_Note: The regex is matched against the full name, which is a combination of the
+test name and all its surrounding suites._
```

Comment:
We've been referring to "suite" as the whole test file - is there a better word here? Is this really just the surrounding describes?

Reply:
Oh, I see.

This is a bit confusing because in some places we do refer to them as ["test suites"](https://facebook.github.io/jest/docs/en/api.html#describename-fn)

I don't mind changing to "describe blocks", but we'll also need to change the main description:

> Run only tests and test suites with a name that matches the regex.

to something like:

Run only tests and describe blocks with a name that matches the regex.

What do you say?

Reply:
From the link:

Reply:
From the link:

> describe(name, fn) creates a block that groups together several related tests in one "test suite". 

doesn't this mean "creates a block that groups together several related tests in one "file"

Reply:
yes, you're right.

I'll change it 😄